cli/command/system/prune.go

```diff
@@ -42,7 +43,7 @@ func newPruneCommand(dockerCli command.Cli) *cobra.Command {
 
 	flags := cmd.Flags()
 	flags.BoolVarP(&options.force, "force", "f", false, "Do not prompt for confirmation")
-	flags.BoolVarP(&options.all, "all", "a", false, "Remove all unused images not just dangling ones")
+	flags.BoolVarP(&options.all, "all", "a", false, "Remove all unused images not just dangling ones, as well as all internal build cache")
```

Comment:
if `--all` is not set, do we prune any build cache at all?

(thinking about slightly different phrasing)